libqtile/config.py

```diff
@@ -287,17 +287,21 @@ def dheight(self):
     def get_rect(self):
         return ScreenRect(self.dx, self.dy, self.dwidth, self.dheight)
 
+    @property
+    def current_group(self):
+        return self.group
```

Comment:
Why this property?

Reply:
I think it's better not to expose the member `self.group` directly, just like [current_group()](https://github.com/qtile/qtile/blob/master/libqtile/core/manager.py#L525) in `manager.py`.

Reply:
Why? `current_group` exists in the manager because the manager doesn't have a direct relation with groups. It's not the case here, a screen has to have a group.

Reply:
You're right, I am removing it now.